### PR TITLE
[Tokens] Add Checkbox color tokens to components

### PR DESCRIPTION
**Pull request by TimRoe**
## Description of Change

Updated components to use tokens 0.17.0 containing the new blues for Checkbox. Updated the Colors Storybook doc page with the new colors.

### Screenshots/Video

image

image

## Testing

- Tested on iOS: N/A 
- Tested on Android: N/A 
- [x] Tested on Web

## PR Checklist
Code reviewer validation:
- General
	- [x] PR is linked to ticket(s)
	- [x] PR has `changelog` label applied if it's to be included in the changelog
	- [x] Acceptance criteria: 
		- All satisfied _or_
		- Documented reason for not being performed _or_
		- Split to separate ticket and ticket is linked by relevant AC(s)
	- [x] Above PR sections adequately filled out
	- [x] If any breaking changes, in accordance with the [pre-1.0.0 versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library#versioning-policy): a CU ticket has been created for the VA Mobile App detailing necessary adjustments with the package version that will be published by this ticket
- Code
	- [x] Tests are included if appropriate (or split to separate ticket)
	- [x] New functions have proper TSDoc annotations

## Publish

While changes warrant a new version [per the versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library#versioning-policy), it is not needed until Checkbox ships so will be deferred.

### CODE REVIEW

File: packages/components/storybook/colors.stories.mdx [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { Meta, ColorPalette, ColorItem } from '@storybook/blocks'
 <Meta title="Design Tokens/Colors" />
 
 # Colors
+
```
Comment (TimRoe) [inline packages/components/storybook/colors.stories.mdx:7]: Autoformatting from VS code added the new lines + changed double quotes to single.

**Comment by TimRoe** [comment]
@jessicawoodin Pulling into this PR to validate the new tokens are placed appropriately within the Forms section (just added at the bottom of the existing Forms colors with the Surface heading).

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Code reviewer validation:
 
 ## Publish
 <!-- Most changes entail a version increment; section can be removed for PRs exclusively within non-ship-relevant files (e.g. unit tests, Storybook stories) -->
-If changes warrant a new version [per the versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library#versioning-policy) and the PR is approved and ready to merge:
+If changes warrant a new version [per the versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library/blob/main/documentation/versioning.md) and the PR is approved and ready to merge:
```
Comment (TimRoe) [inline .github/pull_request_template.md:61]: Noticed this no longer worked correctly, updated link to correct place.

**Comment by jessicawoodin** [comment]
> @jessicawoodin Pulling into this PR to validate the new tokens are placed appropriately within the Forms section (just added at the bottom of the existing Forms colors with the Surface heading).

Looks good to me!

**Review by narin — APPROVED** [review]

**Comment by jessicawoodin** [comment]
@TimRoe While I was updating the tokens in Figma, I realized we didn't add the two new primitive USWDS tokens (blue-warm-5, blue-warm-50) to Storybook. They should appear in the Blue section under USWDS. Since this hasn't been merged yet, would you be able to make that change?

**Comment by TimRoe** [comment]
> @TimRoe While I was updating the tokens in Figma, I realized we didn't add the two new primitive USWDS tokens (blue-warm-5, blue-warm-50) to Storybook. They should appear in the Blue section under USWDS. Since this hasn't been merged yet, would you be able to make that change?

Should appear how?

- With the USWDS BlueVivids?
- A separate USWDS?
- Are we already missing a bunch? I see 15 blues in storybook, but the color tokens themselves appear to have 33 USWDS blues--are we just not using a bunch of the others?

**Comment by jessicawoodin** [comment]
> > @TimRoe While I was updating the tokens in Figma, I realized we didn't add the two new primitive USWDS tokens (blue-warm-5, blue-warm-50) to Storybook. They should appear in the Blue section under USWDS. Since this hasn't been merged yet, would you be able to make that change?
> 
> Should appear how?
> 
> * With the USWDS BlueVivids?
> * A separate USWDS?
> * Are we already missing a bunch? I see 15 blues in storybook, but the color tokens themselves appear to have 33 USWDS blues--are we just not using a bunch of the others?

Yes, it'll appear with the other uswdsSystemColor tokens. The order could be BlueVivid80, BlueVivid30, BlueWarm80, BlueWarm5.
Screenshot 2024-09-18 at 3 19 17 PM

I've only been adding the tokens that we're actively using. VADS has a lot more tokens from USWDS that end up in our tokens package (but we don't actually use yet). For documentation purposes, I decided to focus only on the tokens we actually use.

**Comment by TimRoe** [comment]
Updated:
image

**Pull request merged**